.ci/scripts/build_dswx_ni.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+set -e
+
+# Source the build script utility functions
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+. "${SCRIPT_DIR}"/util.sh
+
+# Parse args
+parse_build_args "$@"
+
+echo '
+=====================================
+
+Building DSWx-NI PGE docker image...
+
+=====================================
+'
+
+PGE_NAME="dswx_ni"
+IMAGE="opera_pge/${PGE_NAME}"
+BUILD_DATE_TIME=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
+
+# defaults, SAS image should be updated as necessary for new image releases from ADT
+[ -z "${WORKSPACE}" ] && WORKSPACE=$(realpath $(dirname $(realpath $0))/../..)
+[ -z "${TAG}" ] && TAG="${USER}-dev"
```

Comment:
for the new directory structure:
[ -z "${WORKSPACE}" ] && WORKSPACE=$(realpath $(dirname $(realpath $0))/../../..)